Source/Core/Core/ConfigManager.h

```diff
@@ -95,7 +95,7 @@ struct SConfig : NonCopyable
   bool bDSPThread = false;
   bool bDSPHLE = true;
   bool bSyncGPUOnSkipIdleHack = true;
-  bool bForceNTSCJ = false;
+  bool bSelectNTSCAsJapanese = false;
```

Comment:
What do you think about `NTSCJInVI` (in the INI)? I was considering something like `NTSCJVIBit`, but it's too hard to see where the word boundary in NTSCJVI is supposed to be. The "VI" in the name probably won't mean anything to users, which isn't that great, but I can't think of another way to specify that it doesn't apply to all regional aspects of the console.